composables/useAuth.ts

```diff
@@ -43,6 +47,7 @@ export function useAuth () {
           evm_wallet: wallet.value
         })
       }
+      loginStatus.value = $t('auth_state.success')
```

Comment:
What if error? How is `loginStatus.value` reset?

Reply:
[🚸 Update blocking modal title and reset login status after authentica…](https://github.com/likecoin/nft-book-press/pull/430/commits/4fc50fe92b228d1922aa8d2b3d5217409d19b8b5)

Close LoginPanel & BlockingModal, show toast error